### DISCUSSION TITLE
Using the host buildx builder instance

### DISCUSSION THREAD

**Discussion by Hentioe** [woodpecker-ci/woodpecker#3857] [Ask a Question]
This is an issue I created on the [buildx plugin](https://codeberg.org/woodpecker-plugins/docker-buildx/issues/169). I want more people to see it, so I post it here again:

My buildx builder instance is relatively complex. It creates multiple nodes and distributes different arch target to different nodes.

I looked at the source code of buildx plugin and it seems to be a fixed `buildx create --use` command (as well as `--config` and `--driver-opt` parameters). It cannot meet my requirements, and I don't have any good ideas to modify the plugin to meet my requirements.

I would like to ask, is there any way to let buildx directly use the host's builder instance? In this case, just delete the `buildx create --use` command.

**Comment by pat-s** [comment]
Please do not [cross-post](https://codeberg.org/woodpecker-plugins/docker-buildx/issues/169).
